common/src/main/scala/explore/components/graphql/LiveQueryRenderMod.scala

```diff
@@ -89,67 +86,6 @@ object LiveQueryRenderMod {
             State.apply
           )
       )
-      .componentDidMount { $ =>
```

Comment:
All this was abstracted away to `Render` in the last refactor, but I forgot to remove it from here. This was causing duplicate queries.